custom_components/google_home/models.py

```diff
@@ -1,7 +1,9 @@
 """Models for Google Home"""
 
+from __future__ import annotations
```

Comment:
This is for using forward declared types in type hints: https://www.python.org/dev/peps/pep-0563/

Reply:
hmmmm, dont really follow what it's going to do 🤔

Reply:
Basically this allows you to use types before they're defined.
For example here it is:

```python3
def get_next_alarm(self) -> Optional[GoogleHomeAlarm]:
```

Otherwise you need to use string:

```python3
def get_next_alarm(self) -> Optional["GoogleHomeAlarm"]:
```

Should be enabled by default in Python 3.10.

Reply:
uuu that't nice, thanks, i hope i get it a bit better now